source/ShipyardPanel.cpp

```diff
@@ -306,7 +306,13 @@ void ShipyardPanel::Sell(bool toStorage)
 
 	int count = playerShips.size();
 	int initialCount = count;
-	string message = "Sell the ";
+	string message;
+	if(!toStorage)
+		message = "Sell the ";
+	else if(count == 1)
+		message = "Sell the hull of the ";
+	else
+		message = "Sell the hulls of the ";
```

Comment:
I saw some places using (outfitsToSell > 1 ? "s" : "") but once again not required, idk if there is a standard

Reply:
It baroque.